packages/core/crates/core-packet/src/packet.rs

```diff
@@ -68,10 +69,41 @@ fn remove_padding(msg: &[u8]) -> Option<&[u8]> {
     Some(&msg.split_at(pos).1[PADDING_TAG.len()..])
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct MixerPayload {
```

Comment:
Wouldn't `TransportData` be a better fitting name? So we have `ApplicationData` on one side, and `TransportData` (or `TransportPayload`) on the other side.

Also if it's used across the crates we can consider putting it into `core-types` `protocol` module.

Reply:
I wanted to avoid too many items called `Data`, because it conveys little information.

Yes, the pr #5427 contains the `core-protocol` package where these public facing types should preferably be aggregated along with the code that works over them and ensures inter-operation with the internal types.